app/main.py

```diff
@@ -1 +1,44 @@
-#  write your code here
+class Animal:
+    def __init__(
+            self,
+            name: str,
+            appetite: int,
+            is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        return 0
+
+
+class Cat(Animal):
+    def __init__(self, name: str, is_hungry: bool = True) -> None:
+        super().__init__(name, appetite=3, is_hungry=True)
+        self.is_hungry = is_hungry
+
+    @staticmethod
+    def catch_mouse() -> None:
+        print("The hunt began!")
+
+
+class Dog(Animal):
+    def __init__(self, name: str, is_hungry: bool = True) -> None:
+        super().__init__(name, appetite=7, is_hungry=True)
```

Comment:
Do not assign `True` to `is_hungry`. 
Just pass it from `__init__`

Do the same for above

Reply:
![Uploading Screenshot 2024-03-28 153928.png…]()

Reply:
![Screenshot 2024-03-28 153928](https://github.com/mate-academy/py-feed-animals/assets/137084131/f8290dae-0627-4737-b831-7dc22960bc6b)

Reply:
![Screenshot 2024-03-28 154551](https://github.com/mate-academy/py-feed-animals/assets/137084131/d8d0c66a-a76c-4a84-9845-6f468f9987ba)